java/src/jmri/jmrix/loconet/locoio/LocoIOData.java

```diff
@@ -541,7 +545,6 @@ protected synchronized void replyReceived() {
             default:
                 log.error("Pin {} unexpected write state, can't advance {}", currentPin, writeState[currentPin]); // NOI18N
                 writeState[currentPin] = NONE;
-                return;
```

Comment:
Maybe this `return` should be restored as well?

Reply:
Done, thanks for spotting